src/integration/wrapper.go

```diff
@@ -69,7 +69,12 @@ func WithHTTPSink(sinkConfig config.HTTPSink) OptionFunc {
 			}
 		}
 
-		prober, err := prober.New(iw.probeTimeout, iw.probeBackoff, prober.WithLogger(iw.logger), prober.WithClient(client))
+		probeTimeout := sinkConfig.ProbeTimeout
+		// if the ProbeTimeout is not set in HTTPSink, just use the default probe timeout from the wrapper
+		if probeTimeout == 0 {
+			probeTimeout = iw.probeTimeout
+		}
```

Comment:
can you remove the `probeTimeout` on the Wrapper struct in this file?
also remove `defaultProbeTimeout` from this file.

it seems like those options only apply to the HTTPSink, and cannot be overridden anyway. so it's strange that they are defined in the wrapper struct.

as a followup, we should also move the probe backoff into an HTTPSink option and remove it from the wrapper.